cuda/test/base/exception_helpers.cu

```diff
@@ -33,12 +33,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/base/exception_helpers.hpp>
 
 
-#include <gtest/gtest.h>
-
-
 #include <cublas_v2.h>
 #include <cuda_runtime.h>
 #include <cusparse.h>
+#include <gtest/gtest.h>
```

Comment:
I understand that these are third party libraries and hence are bunched. But maybe it is better if `gtest/gtest.h` is separate ?

Reply:
If we need to separate them, I prefer to split this section to device-related library (cuda/hip/omp) and other third party libraries.
Otherwise, I would like to keep them together.

Reply:
Yes, device related into one and third party into one would be good.